pkg/controller/br_fed_config.go

```diff
@@ -59,7 +59,7 @@ func DefaultBrFedCLIConfig() *BrFedCLIConfig {
 		LeaseDuration:  15 * time.Second,
 		RenewDeadline:  10 * time.Second,
 		RetryPeriod:    2 * time.Second,
-		ResourceLock:   resourcelock.EndpointsResourceLock,
```

Comment:
not accessible since https://github.com/kubernetes/kubernetes/pull/106852/files#diff-a25ac8933468a654f5d22934e006e00f17a64fd414fb2371540b122d661c4b1eR34